test/lib/subgraphPoolDataService.ts

```diff
@@ -3,6 +3,7 @@ import { PoolDataService, SubgraphPoolBase } from '../../src';
 import { getOnChainBalances } from './onchainData';
 import { Provider } from '@ethersproject/providers';
 
+// @todo?
```

Comment:
You can add your pool specific fields here which I believe will be:
delta
epsilon
and fxRate